app/src/main/java/com/google/droidjump/GameView.java

```diff
@@ -132,6 +167,12 @@ public void drawScene() {
             // Drawing droid
             drawDroid(canvas);
 
+            // Drawing Obstacles
+            drawObstacles(canvas);
+
+            // Drawing platform
+            drawPlatform(canvas);
```

Comment:
nit: I would remove the comments here. The methods seem to be pretty explanatory. Also remove the new lines